demo/site/src/header/Header.tsx

```diff
@@ -66,14 +66,18 @@ const TopLevelLinkContainer = styled.li`
     }
 `;
 
-const Link = styled.a<{ $active: boolean }>`
+const Link = styled(PageLink)`
     text-decoration: none;
     padding: 5px 10px;
-    color: ${({ $active, theme }) => ($active ? theme.colors.primary : theme.colors.black)};
+    color: ${({ theme }) => theme.colors.black};
 
     &:hover {
         text-decoration: underline;
     }
+
+    &.active {
+        color: ${({ theme }) => theme.colors.primary};
+    }
```

Comment:
I achieved the active styling using a plain old class name that's set on the PageLink component. Another solution would be to remove the active check in the component and add an `active` prop that's set from outside. What do you prefer?

Reply:
Add activeClassName as discussed.